plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -2000,5 +2002,40 @@ var _ = Describe("validator", func() {
 				Expect(err).To(HaveOccurred())
 			})
 		})
+
+		Context("shoot deletion", func() {
+			var (
+				shootStore             cache.Store
+				intCoreInformerFactory intCoreinformers.SharedInformerFactory
```

Comment:
```suggestion
				coreInformerFactory coreinformers.SharedInformerFactory
```

Though, why do you need this? There is already the `coreInformerFactory`?

Reply:
Nice catch, tanks! I guess it was just a copy-paste mistake